geo-types/Cargo.toml

```diff
@@ -22,6 +23,7 @@ serde = { version = "1", optional = true, features = ["derive"] }
 # rstar integration relies on the optional approx crate, but implicit features cannot yet enable other features.
 # See: https://doc.rust-lang.org/nightly/cargo/reference/unstable.html#namespaced-features
 rstar = { version = "0.8", optional = true }
```

Comment:
so, if/when we eventually make a major update to geo-types, we'd presumably drop the older rstars?

Reply:
Yes, we can drop old versions based on some agreeable policy; simplest would be to drop all but latest.

Another question is we adopt a uniform "use-{crate}-{version}" name for the feature or if there's going to be one without version.  I think the uniform naming makes more sense, but don't have a strong opinion.

Reply:
I'd say uniform. I didn't add a comment about it because it would imply a breaking change in the next release, which I assume you wanted to avoid.

Reply:
Makes sense. Will add some docs so we adopt a policy at the next breaking release.